vllm/model_executor/model_loader/neuron.py

```diff
@@ -108,20 +106,8 @@ def load_weights(self, model_name_or_path: str, **kwargs):
         neuronx_module = importlib.import_module(neuronx_module_path)
         neuronx_model_cls = getattr(neuronx_module, neuronx_model_cls_name)
 
-        split_model_dir = f"{model_name_or_path}-split"
-        if _is_pretrained_neuron_checkpoint(model_name_or_path):
-            split_model_dir = model_name_or_path
-        elif not os.path.exists(f"{model_name_or_path}-split"):
-            hf_model_cls = getattr(transformers, hf_model_cls_name)
-            from transformers_neuronx.module import save_pretrained_split
-
```

Comment:
we can still leverage the saved split and don't have to remove the code, the support for safe tensor checkpoint is added.

Reply:
ignore saw it's not needed after 2.18 (https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/transformers-neuronx/transformers-neuronx-developer-guide.html#checkpoint-support-and-automatic-model-selection)